pkg/sink/cloudevent/cloudevent.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2022 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cloudevent
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	cloudevents "github.com/cloudevents/sdk-go/v2"
+	"go.uber.org/zap"
+
+	triggersv1 "github.com/tektoncd/triggers/pkg/apis/triggers/v1beta1"
+)
+
+// CEClient matches the `Client` interface from github.com/cloudevents/sdk-go/v2/cloudevents
+type CEClient cloudevents.Client
+
+// SendCloudEvents is used by the EventListener to send cloud events status of
+// trigger processing
+func SendCloudEvents(payload []byte, el triggersv1.EventListener, eventID, eventType, targetURI string, ceClient CEClient, logger *zap.SugaredLogger) {
```

Comment:
For this function we are passing more arguments it looks very big can we make use of structure or something to meet golang coding guidelines